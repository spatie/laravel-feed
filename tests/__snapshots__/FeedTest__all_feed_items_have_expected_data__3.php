<?php return '<?xml version="1.0" encoding="UTF-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
                        <id>http://localhost/feedBaseUrl/feed3</id>
                                <link href="http://localhost/feedBaseUrl/feed3"></link>
                                <title><![CDATA[Feed 3]]></title>
                                <description>This is feed 3 from the unit tests</description>
                                <language>en-US</language>
                                <updated>Fri, 01 Jan 2016 00:00:00 +0100</updated>
                        <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>Fri, 01 Jan 2016 00:00:00 +0100</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>Fri, 01 Jan 2016 00:00:00 +0100</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>Fri, 01 Jan 2016 00:00:00 +0100</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>Fri, 01 Jan 2016 00:00:00 +0100</updated>
        </entry>
            <entry>
            <title><![CDATA[feedItemTitle]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name> <![CDATA[feedItemAuthor]]></name>
            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                          <enclosure url="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category type="html">
                <![CDATA[feedItemCategory]]>
            </category>
                        <updated>Fri, 01 Jan 2016 00:00:00 +0100</updated>
        </entry>
    </feed>
';
