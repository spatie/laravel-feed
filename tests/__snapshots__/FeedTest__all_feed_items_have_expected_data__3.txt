<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet href="/vendor/feed/atom.xsl" type="text/xsl"?>
<feed xmlns="http://www.w3.org/2005/Atom" xml:lang="en-US">
                        <id>http://localhost/feedBaseUrl/feed3</id>
                                <link href="http://localhost/feedBaseUrl/feed3" rel="self"></link>
                                <title><![CDATA[Feed 3]]></title>
                                <logo>http://localhost/image.jpg</logo>
                                <subtitle>This is feed 3 from the unit tests</subtitle>
                                                    <updated>2015-12-31T22:59:00+00:00</updated>
                        <entry>
            <title><![CDATA[feed<>]]&gt;Item"Title"]]></title>
            <link rel="alternate" href="https://localhost/news/testItem1" />
            <id>http://localhost/1</id>
            <author>
                <name><![CDATA[feedItemAuthor]]></name>
                <email><![CDATA[feedItemAuthor@test.test]]></email>

            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                        <link href="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category term="feedItemCategory" />
                        <updated>2015-12-31T22:59:00+00:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feed<>]]&gt;Item"Title"]]></title>
            <link rel="alternate" href="https://localhost/news/testItem2" />
            <id>http://localhost/2</id>
            <author>
                <name><![CDATA[feedItemAuthor]]></name>
                <email><![CDATA[feedItemAuthor@test.test]]></email>

            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                        <link href="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category term="feedItemCategory" />
                        <updated>2015-12-31T22:58:00+00:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feed<>]]&gt;Item"Title"]]></title>
            <link rel="alternate" href="https://localhost/news/testItem3" />
            <id>http://localhost/3</id>
            <author>
                <name><![CDATA[feedItemAuthor]]></name>
                <email><![CDATA[feedItemAuthor@test.test]]></email>

            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                        <link href="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category term="feedItemCategory" />
                        <updated>2015-12-31T22:57:00+00:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feed<>]]&gt;Item"Title"]]></title>
            <link rel="alternate" href="https://localhost/news/testItem4" />
            <id>http://localhost/4</id>
            <author>
                <name><![CDATA[feedItemAuthor]]></name>
                <email><![CDATA[feedItemAuthor@test.test]]></email>

            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                        <link href="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category term="feedItemCategory" />
                        <updated>2015-12-31T22:56:00+00:00</updated>
        </entry>
            <entry>
            <title><![CDATA[feed<>]]&gt;Item"Title"]]></title>
            <link rel="alternate" href="https://localhost/news/testItem5" />
            <id>http://localhost/5</id>
            <author>
                <name><![CDATA[feedItemAuthor]]></name>
                <email><![CDATA[feedItemAuthor@test.test]]></email>

            </author>
            <summary type="html">
                <![CDATA[feedItemSummary]]>
            </summary>
                        <link href="http://localhost/image1.jpg" length="31300" type="image/jpeg" />
                                    <category term="feedItemCategory" />
                        <updated>2015-12-31T22:55:00+00:00</updated>
        </entry>
    </feed>
